.github/workflows/ci.yml

```diff
@@ -170,7 +170,7 @@ jobs:
 
     - name: Run integration tests
       run: |
-        sudo timeout -k 30 1h python3 -m pytest \
+        sudo --preserve-env=GITHUB_ACTIONS timeout -k 30 1h python3 -m pytest \
```

Comment:
This can just be --preserve-env, we don't care about security in CI

Reply:
Changed it to a plain `--preserve-env`